devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy

```diff
@@ -21,7 +21,7 @@ metadata {
 
 		attribute "alarmState", "string"
 
-		fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
```

Comment:
@parasthakkar-newell  are you sure you want to remove support for old model of first alert . If you remove this fingerprint those devices that are reporting model 0002 will stop working with SmartThings platform. Maybe just adding new fingerprint is enough.

Reply:
> 
> Mariusz, I am not asking to remove support for current device (", model:"0002"). I am requesting addition of new device (", model:"0003"). Both the device will be in the market at the same time. Thanks, Paras From: Mariusz A <notifications@github.com> Sent: Thursday, November 21, 2019 7:26 AM To: SmartThingsCommunity/SmartThingsPublic <SmartThingsPublic@noreply.github.com> Cc: Thakkar, Paras <Paras.Thakkar@Newellco.com>; Mention <mention@noreply.github.com> Subject: Re: [SmartThingsCommunity/SmartThingsPublic] DevWs for FirstAlert containing containing Z-Wave Smoke Alarm (#12335) @MAblewiczS requested changes on this pull request. @parasthakkar-newell<https://github.com/parasthakkar-newell> check my note about removing old fingeprint and give me confimation. New fingerprint is ok.
> ________________________________ In devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy<[#12335 (comment)](https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/12335#discussion_r349081655)>:
> @@ -21,7 +21,7 @@ metadata {
> attribute "alarmState", "string" - fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)" @parasthakkar-newell<https://github.com/parasthakkar-newell> are you sure you want to remove support for old model of first alert . If you remove this fingerprint those devices that are reporting model 0002 will stop working with SmartThings platform. Maybe just adding new fingerprint is enough. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<#12335?email_source=notifications&email_token=AKMJLXLJKTNIDSXDGPKYGCLQU2D7DA5CNFSM4JPULJ2KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCMQDOTA#pullrequestreview-320878412>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKMJLXJMN3BT5F3SQ5DBVJDQU2D7DANCNFSM4JPULJ2A>.

So you need to leave old Fingeprint intact (i mean line fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)") and add new line with new fingeprint. Curently from Pull request it looks like you are replacing old Fingeprint with new one (by deleting old line and adding new one).

Reply:
Mariusz,
I have updated branch parasthakkar-newell:firstalert-zcombo-smoke-alarm_1 and added deleted line back. Please check the branch and confirm that this will have support for both models.

Reply:
I checked you branch. It looks ok now. Create PR and let me know (you may add me as reviewer)

Reply:
@MAblewiczS  I have made same changes to branch devws-for-firstalert-containing_2 and same pull request - Please review.